org.jacoco.core.test.validation.java14/src/org/jacoco/core/test/validation/java14/SwitchExpressionsTest.java

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2019 Mountainminds GmbH & Co. KG and Contributors
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Evgeny Mandrikov - initial API and implementation
```

Comment:
@marchof copy-paste? 😉 that's [one of the reasons why I think that `Contributors` field doesn't make sense nowadays](https://github.com/jacoco/jacoco/pull/943#issuecomment-533833196)

Reply:
Grrr, yep let's remove them soon. Fixed.